### DISCUSSION TITLE
Routes in a deployed network

### DISCUSSION THREAD

**Discussion by wdargie** [contiki-ng/contiki-ng#2459] [General]
We deployed RE (revised b) nodes around a lake at FIU to collect some statistics. We are using RPL LIte and wish to print all the routes in the netqwork in order to associate the stats with  individual nodes. Despite the apparent presence of multiple routes UIP_MAX_ROUTES is evaluated zero in:

#if (UIP_MAX_ROUTES != 0)
        PRINTF("Routing entries: %u\n", uip_ds6_route_num_routes());
      #endif
      #if (UIP_SR_LINK_NUM != 0)
        PRINTF("Routing links: %u\n", uip_sr_num_nodes());
      #endif
The number of links are displayed correctly, though. 

Has anyone any tip?

**Comment by pjonsson** [comment]
I don't know RPL Lite, but the `UIP_MAX_ROUTES` definition in `os/net/ipv6/uip-ds6-route.h` says "RPL Lite only supports non-storing, no routes".
